bitcoind/btcjson/help_test.go

```diff
@@ -565,7 +565,7 @@ func TestHelpArgInternals(t *testing.T) {
 
 // TestMethodHelp ensures the method help function works as expected for various
 // command structs.
-func TestMethodHelp(t *testing.T) {
```

Comment:
What is that? Could we give it more meaningful name?

Reply:
fixed